lib/hello_world.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "hello_world.h"
+
+std::string HelloWorld::get_hello_world_string() {
+    return "Hello World!";
```

Comment:
Just saw this now, but I think tab size should be 2 spaces 
https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs

Reply:
Fixed it!